gordo_components/cli/workflow_generator.py

```diff
@@ -118,6 +118,13 @@ def workflow_cli():
     help="The docker registry to use for pulling component images from",
     envvar=f"{PREFIX}_DOCKER_REGISTRY",
 )
+@click.option(
+    "--log-level",
+    type=str,
+    default=None,
+    help="Run workflow with custom log-level.",
+    envvar=f"{PREFIX}_LOG_LEVEL",
```

Comment:
Now it is a bit confusing, since there is both `LOG_LEVEL` and `{PREFIX}_LOG_LEVEL`, one influencing the running instance and the other only setting the variable for generated workflow. Also, the workflow has the environment variable as `GORDO_LOG_LEVEL`, but that variable is no longer read:-p